.github/workflows/tests.yml

```diff
@@ -11,6 +11,9 @@ on:
     - cron: "23 3 */3 * *"
 jobs:
   run_tests:
+    permissions:
```

Comment:
Question: the token is needed for workload identity auth?

Reply:
Yes, exactly.

https://github.com/marketplace/actions/authenticate-to-google-cloud#usage